### PR TITLE
added some v4L2 constants

### PR DESCRIPTION
**Pull request by achatto**

### CODE REVIEW

File: src/c.rs [hunk 1]
```diff
@@ -67,6 +67,9 @@ pub const V4L2_CTRL_CLASS_USER: u32 = 9961472;
 pub const V4L2_CID_BASE: u32 = 9963776;
 pub const V4L2_CID_ARDUCAM_BASE: u32 = 9967872;
 pub const V4L2_CID_ARDUCAM_EXT_TRI: u32 = 9967873;
+pub const V4L2_CID_VFLIP: u32 = 9963797;//
+pub const V4L2_CID_HFLIP: u32 = 9963796;//
+pub const V4L2_CID_GAIN: u32 = 9963795;//
```
Comment (m-ou-se) [inline src/c.rs:72]: Why are there empty comments (`//`) after these?

Comment (achatto) [reply src/c.rs:72]: oh! I intended to remove them before my commit

Comment (m-ou-se) [reply src/c.rs:72]: You can edit your commit with `git commit --amend` (after adding your changes), and then force push with `git push -f`.

Comment (achatto) [reply src/c.rs]: Thanks. I update the PR

**Review by m-ou-se — APPROVED** [review]

**Comment by m-ou-se** [comment]
Published as version 0.1.1.

**Pull request merged**